pkg/hintrunner/zero/zerohint_keccak.go

```diff
@@ -407,6 +407,45 @@ func createKeccakWriteArgsHinter(resolver hintReferenceResolver) (hinter.Hinter,
 	return newKeccakWriteArgsHint(inputs, low, high), nil
 }
 
+// CompareKeccakFullRateInBytes hint compares a value to KECCAK_FULL_RATE_IN_BYTES constant, i.e., 136
+//
+// `newKeccakWriteArgsHint` takes 1 operander as argument
+//   - `nBytes` is the value to be compared with KECCAK_FULL_RATE_IN_BYTES
+//
+// `newKeccakWriteArgsHint` writes 1 or 0 to `ap` memory address depending on whether
+// `n_bytes` is greater or equal to KECCAK_FULL_RATE_IN_BYTES or not
+func newCompareKeccakFullRateInBytesHint(nBytes hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "CompareKeccakFullRateInBytes",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
+			//> ids.n_bytes >= ids.KECCAK_FULL_RATE_IN_BYTES
+			nBytesVal, err := hinter.ResolveAsUint64(vm, nBytes)
```

Comment:
Same as previous hint. Should we not resolve this as a felt ? 

From cairo-lang : `n_bytes: felt`

Reply:
mmh,
n_bytes is:

```
n_bytes=n_elements * 32
```

n_elements is a felt, but represents the number of uint256  (or felts) on which we apply the keccak.

It seems logic that n_elements will never be bigger than uint64, but we never know?  and `n_bytes` is `n_elements * 32` , so i guess theoretically we should use felts indeed here?

@cicr99  what do you think?

Reply:
ok will add a comment, thanks @cicr99  !